### ISSUE TITLE
peer_review branch does not retrieve GitHub PAT using gitcreds

### ISSUE THREAD

**Issue by jooyoungseo** [rundel/ghclass#129]
Currently, the `peer_review` branch seems to retrieve GitHub PAT directly from .Renviron file.

I have securely saved my token using `gitcreds::gitcreds_set()` already.

When I tested using `ghclass::github_test_token()`, the `peer_review` version threw an error while the CRAN version and the `main` branch version were able to successfully retrieve the token as expected.

**Comment by jooyoungseo** [comment]
In other words, the `peer_review` version requires `GITHUB_PAT` variable in `.Renviron` file, which is a deprecated approach.

**Comment by rundel** [comment]
Yep unfortunately that is the case currently - I have not had a chance to backport the more recent changes in ghclass to this branch. I will try to do so later this week and hopefully it should go more smoothly.
